src/content/docs/codestream/start-here/codestream-new-relic.mdx

```diff
@@ -0,0 +1,228 @@
+* [A workload for errors inbox](/docs/new-relic-one/use-new-relic-one/workloads/use-workloads/#create)
+* [An error](/docs/apm/apm-ui-pages/errors-inbox/errors-inbox/#error-details)
+* [For APM errors, your repository's commit hash and release tag](#apm)
+
+New Relic errors inbox is a single place to proactively detect, prioritize, and take action on your errors before they impact customers. With CodeStream, you can jump from an error directly to the offending code in your IDE.
+
+<Callout variant="tip">
+    Limited to APM errors.
+</Callout>
+
+From **[one.newrelic.com/](https://one.newrelic.com/)**, go to **Errors Inbox**, click a stack trace error, then click **Open in IDE**.
+
+## APM errors and CodeStream [#apm]
+
+To use CodeStream's **Open in IDE** with your APM stack trace errors, configure your APM agent with your application's [commit sha](https://git-scm.com/book/en/v2/Git-Tools-Revision-Selection)
+and/or your [release tag](https://git-scm.com/book/en/v2/Git-Basics-Tagging) associated with the running version of your software.  
+
+CodeStream only needs the first seven characters of your commit sha (for example, `734713b`) to make this connection, but you can include the entire sha.  
+
+Alternately, you can use a release tag (such as `v0.1.209` or `release-209`) for CodeStream to find the correct version of your code.
+
+For New Relic APM, the commit and/or release tag (`tags.commit` and `tags.releaseTag`) are added as attributes on `Transaction` and `TransactionError` events. The APM agent protocol provides an easy way to set this up via environment variables.  
+
+We recommend setting one or both of these variables as part of your build pipeline.
+* `NEW_RELIC_METADATA_COMMIT` - The commit sha. You can include the whole thing or only the first seven characters.
+* `NEW_RELIC_METADATA_RELEASE_TAG` - A release tag (such as `v0.1.209` or `release-209`). This has the advantage of being human readable.
+
+## Give us your git repository's URL [#repo-url]
+
+In order to link your data in New Relic to your code, we need to know the URL for your git repository.  
+
+The URL can be in either the SSH or HTTPS format:
+* `git@github.com:newrelic/beta-docs-site.git`
+* `https://github.com/newrelic/beta-docs-site.git`
+
+<Callout variant="caution">
+It's possible to add the same GitHub repository more than once, if you're different protocols. The UI warns you about this, but won't prevent you from doing so.
+
+For example, `https://github.com/tuna/repo` and `git@github.com:tuna/repo` are the same repo, with different protocols.
+</Callout>
+
+You can use any one of these methods to set up your repository:
+* Set up via environment variable using New Relic APM. We recommend this method.
+* Use the NerdGraph API.
+
+<CollapserGroup>
```

Comment:
I'd get rid of the list and just go with the collapsers. I was really confused. 
So:
You can use any of these methods to set up your repository:
<CollapserGroup> and so forth. Just add "Recommended" to the "title" for the first one.